lib/graphql/language/lexer.rb

```diff
@@ -1437,6 +1437,9 @@ def self.emit_string(ts, te, meta, block:)
 meta[:previous_token],
 )
 else
+# As the string is modified in place below, duping the value will provide a non-frozen string to modify.
+# `GraphQL::Language::BlockString.trim_whitespace`, can return frozen strings.
+value = value.dup if value.frozen?
```

Comment:
The only thing I'm worried about is it appears in documentation that Lexer is automatically created, would this override what I've defined?

Reply:
That's right -- this file is generated by `lexer.rl`, which is a pain to generate (and getting harder .... Ragel has a new dependency, and its new version generates a slower state machine than the old version 😖 ) 

IMO, better to modify `BlockString.trim_whitespace` for that reason. Was your fix there sufficient to solve your problem?

Reply:
Sounds good @rmosolgo, I'll modify BlockString, and `dup` any instances of frozen strings, as it did solve my problem.

As a note, I did actually resolve the issue by updating the description so it wasn't empty, or by removing it, but thought that this might be something others encounter in the future.